README.md

```diff
@@ -1 +1,1205 @@
+			 \ r-7  `-. ._  ' .  `\
+			  \`,      `-.`7  7)   )
+			   \/         \|  \'  / `-._
+			              ||    .'
+			               \\  (
+			                >\  >
+			            ,.-' >.'
+			           <.'_.''
+			             <'
+
+To git.balena-cloud.com:heds/cliapp.git
+ * [new branch]      initial-pr -> master
+ ```
+
+This will store the code from the local repository into the private copy
+stored by balena.
+
+Just like `balena push`, once the code has been built it will be downloaded
+by the device and then run.
+
+**Important Note:** There is absolutely no guarantee that the private repository
+will be available for pulling code from, and as such it should never be used
+as an application's source repository origin. It is intended to be a temporary
+repository used only for pushing to the builders.
+
+## 4. SSHing into a Device
+
+Once a device has been provisioned, it can be accessed by SSHing into it via
+the balenaClound VPN. To do this, you'll need the UUID of the device you want to
+SSH into (remember you can see all your devices by running `balena devices`):
+```
+$ balena ssh 1234567
+=============================================================
+    Welcome to balenaOS
+=============================================================
+root@827b231:~#
+```
+By default, SSH access is routed into the host balenaOS shell. However, you
+can SSH into a service by specifying its name as part of the command:
+```
+$ balena ssh 1234567 main
+root@827b231:/usr/src/app#
+```
+This also works in multicontainer applications, simply pass the name of the
+appropriate service as defined in `docker-compose.yml` you'd like to access
+the shell for.
+
+When using device UUIDs, `balena ssh` uses the balena VPN to create a secure
+tunnel to the device and then forward SSH traffic between it and your
+development machine (for production devices, this is the only available method).
+
+For devices running development images on your local network, you can also
+use SSH by specifying the hostname or IP address of that device (development
+images have SSH enabled by default). Using `balena ssh` in this way doesn't use
+the balena VPN and instead makes a direct SSH connection to the device.
+For example:
+```
+$ balena ssh 192.168.1.2
+```
+To find the hostname of a local development device, you can use `balena scan`:
+```
+$ sudo balena scan
+Reporting scan results
+-
+  host:          827b231.local
+  address:       192.168.1.173
+  dockerInfo:
+    Containers:        1
+    ContainersRunning: 1
+    ContainersPaused:  0
+    ContainersStopped: 0
+    Images:            2
+    Driver:            aufs
+    SystemTime:        2019-09-05T13:31:34.910619617Z
+    KernelVersion:     4.14.98
+    OperatingSystem:   balenaOS 2.38.0+rev1
+    Architecture:      armv7l
+  dockerVersion:
+    Version:    18.09.6-dev
+    ApiVersion: 1.39
+```
+In this instance `827b231.local` is the hostname, so the device can be SSHd
+into using `balena ssh 827b231.local`.
```

Comment:
Probably worth mentioning the default is `<short-uuid>.local` to save somebody having to do this.

Reply:
Done.